### DISCUSSION TITLE
why EmbassyOS?

### DISCUSSION THREAD

**Discussion by mpkaiser** [Start9Labs/start-os#542] [General]
Why EmbassyOS?  What is so special about it to justify the cost?

**Comment by kn0wmad** [comment]
Hi there, thanks for reaching out!

You might find the FAQs on this page to be helpful - https://docs.start9.com/support/FAQ/general-faq.html#what-is-the-embassy

A simple explanation is that EmbassyOS is a way to use proven technologies that make it easy for individuals to have more sovereignty and independence in their digital lives, without needing lots of technical skill.  These technologies have been around a long time, but we aim to make them accessible to the masses in order to help combat the growing censorship and centralization of power in the tech space.  If you have specific questions, or need me to expound on this further, please don't hesitate to ask! :)
